Front-End/src/Pages.js

```diff
@@ -135,19 +137,22 @@ function Drivetrain(props) {
                         <option value="PLA">PLA</option>
                         <option value="ONYX">ONYX</option>
                     </select>
-                    <br></br>
-
-                    <input type="text" id="LengthInInches" name="drivetrain_LengthInInches" placeholder="Length in Inches" className="text-input" required="required" />
-                    <br></br>
-                    <input type="text" id="PartNumber" name="drivetrain_PartNumber" placeholder="Part Number" className="text-input" required="required" />
-                    <br></br>
-                    <input type="radio" id="Low" name="Low" value="HTML" />
-                    <label for="html">Low</label>
-                    <input type="radio" id="Mid" name="Mid" value="Mid" />
-                    <label for="Mid">Mid</label>
```

Comment:
It looks like the "Length in Inches" and "Part Number" inputs got deleted on the Drivetrain page. Was this on purpose?

Drivetrain:
![image](https://user-images.githubusercontent.com/6099416/222065234-0cd38ae3-f590-408a-84a6-35d287291419.png)


Not drivetrain:
![image](https://user-images.githubusercontent.com/6099416/222065479-ae571fc2-26f4-4b9b-b7e9-cfe4f4e21db7.png)

Reply:
fixed.

Reply:
it is fixed

Reply:
perfect. thanks for fixing!